Odata-docs/webapi-8/fundamentals/overview.md

```diff
@@ -0,0 +1,581 @@
+---
+title: "ASP.NET Core OData 8 Fundamentals Overview"
+description: "This document introduces the fundamental concepts of ASP.NET Core OData library."
+
+author: habbes
+ms.author: clhabins
+
+ms.date: 1/5/2023
+---
+# Fundamentals Overview
+**Applies To**:[!INCLUDE[appliesto-webapi](../../includes/appliesto-webapi-v8.md)]
+
+This article provides an overview of the fundamental concepts for building ASP.NET Core OData apps.
+
+ASP.NET Core OData is a .NET library that sits on top of ASP.NET Core to help you build REST APIs. The ASP.NET Core OData library consists of a set of services and middleware that hook into ASP.NET Core's request pipeline to provide features like routing, query handling and serialization based on OData specifications and conventions. All these concepts will be covered in depth in separate articles, this just provides an overview.
+
```

Comment:
@habbes consider including a list that outlines the concepts that are detailed in the article e.g. 1.create edm model. 2. register services 3.configure querying etc...
i think it would be useful to a complete beginner to OData and even ASP.NET have a logical view of how to set up the various bits for an OData service

Reply:
Do you mean like a list in the introduction, or something like a "table of contents".

If it's the latter, then it's auto-generated:

![image](https://user-images.githubusercontent.com/8460169/211771291-e4f4d374-3b3e-49ed-b84b-38d072f0a597.png)

Reply:
something like this 
![image](https://user-images.githubusercontent.com/7426531/211772785-67039dfc-0db2-4aa5-b5a3-7d649b3f1444.png)

Reply:
I have gone through a few samples in the ASP.NET Core docs and it appears this this outline is mostly used in tutorials-like article where the entire article takes you through a sequence of steps to achieve a goal, e.g.: 
- https://learn.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/razor-pages-start?view=aspnetcore-6.0&tabs=visual-studio
- https://learn.microsoft.com/en-us/aspnet/core/tutorials/first-web-api?view=aspnetcore-6.0&tabs=visual-studio
- https://learn.microsoft.com/en-us/aspnet/core/getting-started/?view=aspnetcore-6.0&tabs=windows

But it's not include in articles that provide a guide on conceptual topics:
- https://learn.microsoft.com/en-us/aspnet/core/fundamentals/?view=aspnetcore-6.0&tabs=windows
- https://learn.microsoft.com/en-us/aspnet/core/fundamentals/servers/?view=aspnetcore-6.0&tabs=windows

This is not a solid argument for not including that part. And I'm not against the suggestion. I'm just not sure what to write, what the exact outline should be. What I have in mind is to list the headings, but that would be a duplicate of the outline already auto-generated in the sidebar. If you think this is fine, I can go ahead and add it.

Reply:
cc @asanyaga